lib/stdlib/src/shell_default.erl

```diff
@@ -75,51 +75,55 @@ code:load_abs("$PATH/user_default").
 -doc "Print the help for all shell internal commands.".
 -spec help() -> true.
 help() ->
-    format("** shell internal commands **~n"),
-    format("b()        -- display all variable bindings\n"),
-    format("e(N)       -- repeat the expression in query <N>\n"),
-    format("f()        -- forget all variable bindings\n"),
-    format("f(X)       -- forget the binding of variable X\n"),
-    format("h()        -- history\n"),
-    format("h(Mod)     -- help about module\n"),
-    format("h(Mod,Func)-- help about function in module\n"),
-    format("h(Mod,Func,Arity) -- help about function with arity in module\n"),
-    format("ht(Mod)    -- help about a module's types\n"),
-    format("ht(Mod,Type) -- help about type in module\n"),
-    format("ht(Mod,Type,Arity) -- help about type with arity in module\n"),
-    format("hcb(Mod)    -- help about a module's callbacks\n"),
-    format("hcb(Mod,CB) -- help about callback in module\n"),
-    format("hcb(Mod,CB,Arity) -- help about callback with arity in module\n"),
-    format("history(N) -- set how many previous commands to keep\n"),
-    format("results(N) -- set how many previous command results to keep\n"),
-    format("catch_exception(B) -- how exceptions are handled\n"),
-    format("v(N)       -- use the value of query <N>\n"),
-    format("rd(R,D)    -- define a record\n"),
-    format("rf()       -- remove all record information\n"),
-    format("rf(R)      -- remove record information about R\n"),
-    format("rl()       -- display all record information\n"),
-    format("rl(R)      -- display record information about R\n"),
-    format("rp(Term)   -- display Term using the shell's record information\n"),
-    format("rr(File)   -- read record information from File (wildcards allowed)\n"),
-    format("rr(F,R)    -- read selected record information from file(s)\n"),
-    format("rr(F,R,O)  -- read selected record information with options\n"),
-    format("lf()       -- list locally defined functions\n"),
-    format("lt()       -- list locally defined types\n"),
-    format("lr()       -- list locally defined records\n"),
-    format("ff()       -- forget all locally defined functions\n"),
-    format("ff({F,A})  -- forget locally defined function named as atom F and arity A\n"),
-    format("tf()       -- forget all locally defined types\n"),
-    format("tf(T)      -- forget locally defined type named as atom T\n"),
-    format("fl()       -- forget all locally defined functions, types and records\n"),
-    format("save_module(FilePath) -- save all locally defined functions, types and records to a file\n"),
+    S = ~"""
```

Comment:
String indent is being added with this change ("         " prefix).

Reply:
No it is not. This is because the ending triple quote decides the indentation level of the string. See https://www.erlang.org/docs/27/system/data_types.html#tqstring for details and examples.

Reply:
Ok, thanks for the link.  Was expecting it to be like Python.